docs/topics/vscode-extension-resolving-issues.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-resolving-issues_{context}']
+
+= Resolving issues
+
+You can resolve issues by using a Quick Fix (automatic code replacement) or by editing the code.
+
+.Procedure
+
+. Resolve issues detected by the {ProductShortName} extension by performing one of the following actions:
+
+* If the issue displays the Quick Fix icon (image:vs_optional.png[Optional or Quick Fix]), right-click the issue and select *Preview Quick Fix*. To accept the suggested fix, right-click the issue and select *Apply Quick Fix.*
+* If the issue has any other icon or you decide not to use all suggested Quick Fixes, right-click the issue, select *Open Code* to open the file involved, make any changes needed to the code, and then save the file.
+
+. Optional: After you fix an issue, you can right-click it and select *Mark as Complete*. The extension displays the Resolved icon (image:vs_resolved.png[Resolved]) next to the issue. This has no effect on the code.
+
+. Optional: To remove an issue from the list, right-click it and select *Delete*. This has no effect on the code.
++
+*Delete* and *Mark as complete* are designed to make it easier for you to keep track of which issues you have finished working with and which still need attention. Files marked deleted or completed will be analyzed again the next time you analyze a project that contains them.
```

Comment:
This section updated on the basis of an email from John Steele -- I forwarded you the email, but here's the relevant part again: 

" ["Mark as Complete" is] for the user to track what they've already migrated/changed or resolved/done. 

When a user right-clicks on a migration item (hint or classification) within the MTA Explorer, and clicks the "Mark as complete" item within the context-menu, all it does is change the icon to a green checkmark. 

This way, they have a way of knowing what they've already looked at or resolved. It doesn't necessarily mean they've fixed the issue. 

They might have looked at the issue and decided it's not something they're concerned about and mark it as complete just so they know they've looked at it. Alternatively, they might just decide to delete the issue.
Marking an issue as complete, or deleting it, doesn't change anything in the code.
For example, if the user marks a bunch of issues as complete, or deletes a bunch of issues, and then re-runs the analysis, those same issues will pop up again. 

It's for the user to better manage and know what they've already done. It's for housekeeping purposes only, and doesn't have any other special meaning. If the IDE gets closed, and then they later come back to it, those items they've marked as complete will still be marked as complete with the green checkmark so they can pick up where they left off."

Reply:
I don't recall the email. What is the context? John's section sounds like it is a response to the issue I submitted about unmarking an issue as "complete". If that's the case, it looks like the issue is not going to be handled. Fine with me. If it's something else, let me know.